src/components/Page/PageSpinner.css

```diff
@@ -7,13 +7,14 @@
 
 .spinner {
   position: absolute;
-  width: 100%;
-  height: 100%;
   top: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
top is already 0, Is bottom 0 getting applied?

Reply:
`top:0, right:0, bottom:0, left: 0` is an alternative to `width: 100%, height 100%`. 
Works for this use-case where something needs to render within a non-static parent—it pushes the bounds to extreme edges